src/Symfony/Component/HttpKernel/Controller/ArgumentResolver/RequestPayloadValueResolver.php

```diff
@@ -56,6 +58,7 @@ public function resolve(Request $request, ArgumentMetadata $argument): iterable
         $payloadMappers = [
             MapQueryString::class => ['mapQueryString', Response::HTTP_NOT_FOUND],
             MapRequestPayload::class => ['mapRequestPayload', Response::HTTP_UNPROCESSABLE_ENTITY],
+            MapUploadedFile::class => ['mapUploadedFile', Response::HTTP_BAD_REQUEST],
```

Comment:
A missing file is like a missing form field in the payload. The code should be 422 too.